### PR TITLE
Xray prompt update and typos

### PR DESCRIPTION
**Pull request by ThibaultLSDC**

### CODE REVIEW

**Review by recursix — DISMISSED** [review]

**Comment by ThibaultLSDC** [comment]
Here is how it looks @recursix 

Initial layout:
image

Submitting with a new output pattern:
image

**Comment by Serega6678** [comment]
@ThibaultLSDC could you please add an requirements.txt file with the precise versions of the libraries you are using. I am facing difficulties with the latest version of browsergym

**Comment by ThibaultLSDC** [comment]
Hello @Serega6678, in the latest commit on this PR, I have removed stale imports to some old webarena checks. Other than that, I had no issue when running the following from a clean conda env:
```shell
pip install -r requirements.txt
pip install -e .
python src/agentlab/experiments/launch_exp.py --exp_config agentlab.agents.generic_agent.exp_configs.generic_agent_test
```
I hope this fixes your issues. If not, could you please provide more details on the problem you are facing ?
Thanks

File: src/agentlab/analyze/agent_xray.py [hunk 1]
```diff
@@ -557,6 +593,30 @@ def update_agent_info():
         return None
 
 
+def submit_action(input_text):
+    global info
+    agent_info = info.exp_result.steps_info[info.step].agent_info
+    chat_messages = deepcopy(agent_info.get("chat_messages", ["No Chat Messages"])[:2])
```
Comment (recursix) [inline src/agentlab/analyze/agent_xray.py:599]: while limiting to the first 2 messages?

Comment (recursix) [reply src/agentlab/analyze/agent_xray.py:599]: In the long run, we may want to experiment with the effect of retry

File: src/agentlab/analyze/agent_xray.py [hunk 2]
```diff
@@ -557,6 +593,30 @@ def update_agent_info():
...
+    assert isinstance(chat_messages[1], BaseMessage), "Messages should be langchain messages"
+
+    chat = ChatOpenAI(name="gpt-4o-mini")
+    chat_messages[1].content = input_text
```
Comment (recursix) [inline src/agentlab/analyze/agent_xray.py:603]: In the long run, we may want to modify the system message or other messages as well. We'll see how we can play with the UI. But for now it's good :)

File: src/agentlab/analyze/agent_xray.py [hunk 3]
```diff
@@ -557,6 +593,30 @@ def update_agent_info():
         return None
 
 
+def submit_action(input_text):
+    global info
+    agent_info = info.exp_result.steps_info[info.step].agent_info
+    chat_messages = deepcopy(agent_info.get("chat_messages", ["No Chat Messages"])[:2])
+    assert isinstance(chat_messages[1], BaseMessage), "Messages should be langchain messages"
+
+    chat = ChatOpenAI(name="gpt-4o-mini")
```
Comment (recursix) [inline src/agentlab/analyze/agent_xray.py:602]: You fixed that to 4o-mini as a first test?

**Review by recursix — APPROVED** [review]
Awesome

**Pull request merged**